client/src/component/Pages.jsx

```diff
@@ -69,32 +69,83 @@ class Pages extends React.Component {
     for (let i = 1; i <= numberOfPages; i += 1) {
       pageArray.push(
         <PageButtonList>
-          <PageButton type="button" onClick={() => handlePageClick(i - 1)}>
+          <PageButton type="button" onClick={() => handlePageClick(i - 1, search)}>
             <NumberInButton>{i}</NumberInButton>
           </PageButton>
         </PageButtonList>,
       );
     }
-    if (pageArray.length <= 4) {
+    if (pageArray.length <= 4 && currentPage === 0) {
       return (
         <PageButtonUL>
+          {selectedPageButton}
+          {pageArray[1]}
+          {pageArray[2]}
+          {pageArray[3]}
           <PageButtonList>
-            <PageButton type="button" onClick={backArrowClicked}>
+            <PageButton type="button" onClick={() => forwardArrowClicked(search)}>
+              {backArrow}
+            </PageButton>
+          </PageButtonList>
+        </PageButtonUL>
+      );
+    }
+    if (pageArray.length <= 4 && currentPage === 1) {
+      return (
+        <PageButtonUL>
+          <PageButtonList>
+            <PageButton type="button" onClick={() => backArrowClicked(search)}>
               {forwardArrow}
             </PageButton>
           </PageButtonList>
           {pageArray[0]}
-          {pageArray[1]}
+          {selectedPageButton}
           {pageArray[2]}
           {pageArray[3]}
           <PageButtonList>
-            <PageButton type="button" onClick={forwardArrowClicked}>
+            <PageButton type="button" onClick={() => forwardArrowClicked(search)}>
+              {backArrow}
+            </PageButton>
+          </PageButtonList>
+        </PageButtonUL>
+      );
+    }
+    if (pageArray.length <= 4 && currentPage === 2) {
+      console.log('page 3');
```

Comment:
```suggestion
```

Reply:
I'm so bad at getting rid of those!

Reply:
I'll search through everything real quick and get rid of those